src/cluster/replication.cc

```diff
@@ -103,6 +108,52 @@ void FeedSlaveThread::checkLivenessIfNeed() {
   }
 }
 
+void FeedSlaveThread::staticReadCallback(bufferevent *bev, void *ctx) {
```

Comment:
how about changing it to `replicaReadCallback`

Reply:
the name is taken: https://github.com/apache/kvrocks/pull/3061/files/6e91bf3edeef57e780ae2af56eb4b845f2b071c5#diff-19253031b45ceabae16926533fceb06c0c38f0b2c7e866e9a1a578434f3ca5a4R118